examples/taskruns/taskrun-github-pr-yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: tekton-pr
+spec:
+  type: pullRequest
+  params:
+  - name: url
+    value: https://github.com/tektoncd/pipeline/pull/100
```

Comment:
i think it could be good to put a comment here just explaining why this PR in particular (e.g. "contained labels with characters that need to be escaped" or something like that)

Reply:
Done.